### PR TITLE
V5 release

### PR DESCRIPTION
**Pull request by kgaonkar6**
#### What scientific question is your analysis addressing?
v5 release

#### What was your approach?
- Updated GTEX v8 + KFNBL + PBTA
GTEx files were downloaded from:
TPM https://storage.googleapis.com/gtex_analysis_v8/rna_seq_data/GTEx_Analysis_2017-06-05_v8_RNASeQCv1.1.9_gene_tpm.gct.gz
expected_count https://storage.googleapis.com/gtex_analysis_v8/rna_seq_data/GTEx_Analysis_2017-06-05_v8_RNASeQCv1.1.9_gene_reads.gct.gz

- Added 2/4 consensus maf file
snv-consensus-plus-hotspots.maf.tsv.gz

- Updated histology file 

#### What GitHub issue does your pull request address?
[#35](https://github.com/PediatricOpenTargets/ticket-tracker/issues/35)

#### Which areas should receive a particularly close look?
Please check if download works 

### Data Release Checklist
- [X] Is the table in doc/data-file-descriptions.md up to date?
- [X] Is doc/data-format.md up to date?
- [X] Is doc/release-notes.md up to date?
- [X] Is download-data.sh up to date?
- [x] Was download-data.sh tested and did it complete without error?

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
 # release notes
-## current release
+## curremt release
```
Comment (logstar) [inline doc/release-notes.md:2]: Typo.

**Review by logstar — COMMENTED** [review]
Thank you for preparing the v5 release!

The `gtex_target_tcga-gene-expression-rsem-tpm-collapsed.polya.rds` has `polya` in the filename, but it also contains `stranded` and `poly-A stranded` RNA libraries. I was wondering whether the `polya` should be removed from the filename. `gtex_target_tcga-gene-expression-rsem-tpm-collapsed.polya.rds` has the same samples as `gtex_target_tcga-gene-counts-rsem-expected_count-collapsed.rds`.

`gtex_target_tcga-gene-expression-rsem-tpm-collapsed.polya.rds` and `gtex_target_tcga-gene-counts-rsem-expected_count-collapsed.rds` are no longer available in v5 release <https://s3.amazonaws.com/kf-openaccess-us-east-1-prd-pbta/open-targets/v5/md5sum.txt>. I downloaded them yesterday.

**Comment by kgaonkar6** [comment]
We decided to remove `gtex_target_tcga-gene-expression-rsem-tpm-collapsed.polya.rds` and `gtex_target_tcga-gene-counts-rsem-expected_count-collapsed.rds` 

They are replaced with updated files `gene-counts-rsem-expected_count-collapsed.rds` and `gene-expression-rsem-tpm-collapsed.rds` which now have [GTEX v8 + KFNBL + PBTA]

I'll update the release note to mention that.

**Review by logstar — APPROVED** [review]
> We decided to remove `gtex_target_tcga-gene-expression-rsem-tpm-collapsed.polya.rds` and `gtex_target_tcga-gene-counts-rsem-expected_count-collapsed.rds`
> 
> They are replaced with updated files `gene-counts-rsem-expected_count-collapsed.rds` and `gene-expression-rsem-tpm-collapsed.rds` which now have [GTEX v8 + KFNBL + PBTA]
> 
> I'll update the release note to mention that.

Got it. Thank you for the quick reply!

**Review by logstar — CHANGES_REQUESTED** [review]
Sorry for changing the review status. I found some additional issues.

- The following files currently cannot be downloaded:

filename | URL
--|--
kfnbl-snv-lancet.vep.maf.gz | https://s3.amazonaws.com/kf-openaccess-us-east-1-prd-pbta/open-targets/v5/kfnbl-snv-lancet.vep.maf.gz
kfnbl-snv-mutect2.vep.maf.gz | https://s3.amazonaws.com/kf-openaccess-us-east-1-prd-pbta/open-targets/v5/kfnbl-snv-mutect2.vep.maf.gz
kfnbl-snv-strelka2.vep.maf.gz | https://s3.amazonaws.com/kf-openaccess-us-east-1-prd-pbta/open-targets/v5/kfnbl-snv-strelka2.vep.maf.gz
kfnbl-snv-vardict.vep.maf.gz | https://s3.amazonaws.com/kf-openaccess-us-east-1-prd-pbta/open-targets/v5/kfnbl-snv-vardict.vep.maf.gz

- `gene-expression-rsem-tpm-collapsed.rds` and `gene-counts-rsem-expected_count-collapsed.rds` cannot be loaded.
```text
> readRDS('../../data/v5/gene-expression-rsem-tpm-collapsed.rds')
Error in readRDS("../../data/v5/gene-expression-rsem-tpm-collapsed.rds") : 
  unknown input format
> readRDS('../../data/v5/gene-counts-rsem-expected_count-collapsed.rds')
Error in readRDS("../../data/v5/gene-counts-rsem-expected_count-collapsed.rds") : 
  unknown input format
```

`gene-expression-rsem-tpm-collapsed.rds` and `gene-counts-rsem-expected_count-collapsed.rds` md5sums match `md5sum.txt`:

```text
bash download-data.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
/Users/yz/research/open_pbta_May19_2021/OpenPedCan-analysis
Checking MD5 hashes...
snv-consensus-plus-hotspots.maf.tsv.gz: OK
WGS.hg38.lancet.300bp_padded.bed: OK
WGS.hg38.lancet.unpadded.bed: OK
WGS.hg38.mutect2.vardict.unpadded.bed: OK
WGS.hg38.strelka2.unpadded.bed: OK
WGS.hg38.vardict.100bp_padded.bed: OK
fusion-arriba.tsv.gz: OK
fusion-starfusion.tsv.gz: OK
gene-counts-rsem-expected_count-collapsed.rds: OK
gene-expression-rsem-tpm-collapsed.rds: OK
histologies.tsv: OK
intersect_cds_lancet_strelka_mutect_WGS.bed: OK
intersect_strelka_mutect_WGS.bed: OK
kfnbl-snv-lancet.vep.maf.gz: FAILED
kfnbl-snv-mutect2.vep.maf.gz: FAILED
kfnbl-snv-strelka2.vep.maf.gz: FAILED
kfnbl-snv-vardict.vep.maf.gz: FAILED
release-notes.md: OK
md5sum: WARNING: 4 computed checksums did NOT match
```

- Whether the `TRUE` and `FALSE` values in `histologies.tsv$EFS_days` should be `1` and `0` respectively? `readr::read_tsv` warnings:

```text
Warning: 5 parsing failures.
  row      col expected actual                            file
32309 EFS_days a double  TRUE  '../../data/v5/histologies.tsv'
32319 EFS_days a double  TRUE  '../../data/v5/histologies.tsv'
33477 EFS_days a double  FALSE '../../data/v5/histologies.tsv'
34783 EFS_days a double  FALSE '../../data/v5/histologies.tsv'
35479 EFS_days a double  FALSE '../../data/v5/histologies.tsv'
```

**Comment by kgaonkar6** [comment]
Thanks for reporting the issues.

```
> readRDS('../../data/v5/gene-expression-rsem-tpm-collapsed.rds')
Error in readRDS("../../data/v5/gene-expression-rsem-tpm-collapsed.rds") : 
  unknown input format
> readRDS('../../data/v5/gene-counts-rsem-expected_count-collapsed.rds')
Error in readRDS("../../data/v5/gene-counts-rsem-expected_count-collapsed.rds") : 
  unknown input format
```
I actually saved these files with write_tsv() so they are tab separated files :/ 
Updating to RDS now. Sorry about that!


```
kfnbl-snv-lancet.vep.maf.gz: FAILED
kfnbl-snv-mutect2.vep.maf.gz: FAILED
kfnbl-snv-strelka2.vep.maf.gz: FAILED
kfnbl-snv-vardict.vep.maf.gz: FAILED
```

We don't need these files now I'll update the md5sum.txt.

**Comment by kgaonkar6** [comment]
I'll ping you here when it's ready for a re-download.

**Comment by kgaonkar6** [comment]
Updated the md5sum.txt for the RDS and missing snv files please try another download when you have a chance. 
We will have to get additional info about the EFS_days in `histologies.tsv`.

**Review by logstar — APPROVED** [review]
Thank you for the quick replies. No worries at all. I can imagine the complexity of a new data release.

All downloaded files have matched md5sums:

```text
Checking MD5 hashes...
snv-consensus-plus-hotspots.maf.tsv.gz: OK
WGS.hg38.lancet.300bp_padded.bed: OK
WGS.hg38.lancet.unpadded.bed: OK
WGS.hg38.mutect2.vardict.unpadded.bed: OK
WGS.hg38.strelka2.unpadded.bed: OK
WGS.hg38.vardict.100bp_padded.bed: OK
fusion-arriba.tsv.gz: OK
fusion-starfusion.tsv.gz: OK
gene-counts-rsem-expected_count-collapsed.rds: OK
gene-expression-rsem-tpm-collapsed.rds: OK
histologies.tsv: OK
intersect_cds_lancet_strelka_mutect_WGS.bed: OK
intersect_strelka_mutect_WGS.bed: OK
release-notes.md: OK
```

`gene-expression-rsem-tpm-collapsed.rds` and `gene-counts-rsem-expected_count-collapsed.rds` can also be loaded with `readRDS`.

**Pull request merged**